src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -456,6 +393,109 @@ def test_channel_format_types(self):
         with nifgen.Session(resource_name='', reset_device=False, options='Simulate=1, DriverSetup=Model:5433 (2CH); BoardType:PXIe') as simulated_session:
             assert simulated_session.channel_count == 2
 
+    # Test doesn't run over gRPC because numpy isn't supported by gRPC.
+    def test_create_waveform_from_numpy_array_float64(self, session):
+        data = numpy.ndarray(10000, dtype=numpy.float64)
+        data.fill(0.5)
+        assert type(session.create_waveform(data)) is int
+
+    # Test doesn't run over gRPC because numpy isn't supported by gRPC.
+    def test_create_waveform_numpy_array_int16(self, session):
+        data = numpy.ndarray(10000, dtype=numpy.int16)
+        data.fill(256)
+        assert type(session.create_waveform(data)) is int
+
+    # Test doesn't run over gRPC because numpy isn't supported by gRPC.
+    def test_create_waveform_numpy_array_int16(self, session):
+        data = numpy.ndarray(10000, dtype=numpy.int16)
+        data.fill(256)
+        assert type(session.create_waveform(data)) is int
```

Comment:
Unless I'm mistaken, you have the same test listed twice in a row, here.

Reply:
Yep, the travis-ci says that it's being redefined: https://app.travis-ci.com/github/ni/nimi-python/jobs/587616933

Reply:
I've removed the duplicate test definition.